samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -202,118 +185,151 @@ InferenceEngine::Blob::Ptr createBlobFromBinary(const std::vector<std::string>&
-    InferenceEngine::Blob::Ptr blob =
-        InferenceEngine::make_shared_blob<T>(tDesc,
-                                             std::make_shared<SharedBlobAllocator<T>>((T*)data, blob_size * sizeof(T)));
-    blob->allocate();
-    return blob;
+    auto tensor = ov::runtime::Tensor(inputInfo.precision, inputInfo.dataShape, ov::runtime::Allocator(allocator));
+    return tensor;
 }
 
 template <typename T, typename T2>
-InferenceEngine::Blob::Ptr createBlobRandom(const benchmark_app::InputInfo& inputInfo,
-                                            T rand_min = std::numeric_limits<uint8_t>::min(),
-                                            T rand_max = std::numeric_limits<uint8_t>::max()) {
-    size_t blob_size =
-        std::accumulate(inputInfo.dataShape.begin(), inputInfo.dataShape.end(), 1, std::multiplies<int>());
-    T* data = new T[blob_size];
+ov::runtime::Tensor createBlobRandom(const benchmark_app::InputInfo& inputInfo,
+                                     T rand_min = std::numeric_limits<uint8_t>::min(),
+                                     T rand_max = std::numeric_limits<uint8_t>::max()) {
+    size_t tensor_size =
+        std::accumulate(inputInfo.dataShape.begin(), inputInfo.dataShape.end(), 1, std::multiplies<size_t>());
+    auto allocator = std::make_shared<SharedBlobAllocator>(tensor_size * sizeof(T));
+    auto data = reinterpret_cast<T*>(allocator->getBuffer());
 
     std::mt19937 gen(0);
     uniformDistribution<T2> distribution(rand_min, rand_max);
-    for (size_t i = 0; i < blob_size; i++) {
+    for (size_t i = 0; i < tensor_size; i++) {
         data[i] = static_cast<T>(distribution(gen));
     }
 
-    InferenceEngine::TensorDesc tDesc(inputInfo.precision, inputInfo.dataShape, inputInfo.originalLayout);
-    InferenceEngine::Blob::Ptr blob =
-        InferenceEngine::make_shared_blob<T>(tDesc,
-                                             std::make_shared<SharedBlobAllocator<T>>(data, blob_size * sizeof(T)));
-    blob->allocate();
-    return blob;
+    auto tensor = ov::runtime::Tensor(inputInfo.precision, inputInfo.dataShape, ov::runtime::Allocator(allocator));
+    return tensor;
 }
 
-InferenceEngine::Blob::Ptr getImageBlob(const std::vector<std::string>& files,
-                                        size_t inputId,
-                                        size_t batchSize,
-                                        const std::pair<std::string, benchmark_app::InputInfo>& inputInfo,
-                                        std::string* filenames_used = nullptr) {
+ov::runtime::Tensor getImageBlob(const std::vector<std::string>& files,
+                                 size_t inputId,
+                                 size_t batchSize,
+                                 const std::pair<std::string, benchmark_app::InputInfo>& inputInfo,
+                                 std::string* filenames_used = nullptr) {
```

Comment:
Can we use `std::string&` instead of pointer?

Reply:
nope, this is optional parameter. Why do you afraid of raw pointers ?